<ui version="4.0">
 <class>NomenclUi</class>
 <widget class="QWidget" name="NomenclUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nomenclature</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="1">
    <widget class="QLineEdit" name="prefixED"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="prefixLA">
     <property name="text">
      <string>Sort &amp;as:</string>
     </property>
     <property name="buddy">
      <cstring>prefixED</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="descrLA">
     <property name="text">
      <string>&amp;Description:</string>
     </property>
     <property name="buddy">
      <cstring>descriptionTE</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="symbolLA">
     <property name="text">
      <string>&amp;Symbol:</string>
     </property>
     <property name="buddy">
      <cstring>symbolED</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="symbolED">
     <property name="toolTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="descriptionTE">
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>symbolED</tabstop>
  <tabstop>descriptionTE</tabstop>
  <tabstop>prefixED</tabstop>
 </tabstops>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
